extensions/interactions/GraphInput/directives/graph-viz.component.html

```diff
@@ -25,6 +26,16 @@
     float: right;
     height: 35px;
   }
+  .accessibility-dot {
```

Comment:
Start all CSS classes added by us with "oppia", to distinguish from third-party classes.

Not sure what "accessibility dot" means, too. Maybe choose a more descriptive name?

Reply:
@seanlip what do you think of: `oppia-dot-to-add-node` (because this dot is only active if the user selects the `add node` option)?

Reply:
I don't really know what "dot-to-add-node" means. Perhaps "oppia-add-node-cursor"?

Reply:
That sounds good, I have done the changes. PTAL, thanks.